src/exprs.c

```diff
@@ -998,9 +998,15 @@ Obj             EvalListTildeExpr (
 **  This two step allocation  is necessary, because  list expressions such as
 **  '[ [1], ~[1] ]'  requires that the value of  one subexpression is entered
 **  into the list value before the next subexpression is evaluated.
+**
+**  'tildeInUse' is 1 when this list is part of the value of ~.
+**
+**  When 'tildeInUse' is 1, use a slower code path which ensures the list is
+**  in a valid state after each element is added and handles the elements
+**  and TNUM of the list changing whenever a new element is constructed.
+**
 */
-Obj ListExpr1 (
-    Expr                expr )
+ALWAYS_INLINE Obj ListExpr1(Expr expr, Int safe)
```

Comment:
The documentation now says `tildeInUse`, great -- but the code still uses `safe` :-)

Reply:
bla!